newsrevenuehub/rev-engine
# Nrh 358  revenue program statement descriptor

#### What's this PR do?
Adds a field to RevenueProgram `stripe_statement_descriptor_suffix`. It has a max size of 10 chars. If it is set, when a PaymentIntent is created it is added as `statement_descriptor_suffix`. If it is not set, nothing is set on the Intent, and the statement descriptor will be the Account level `statement_descriptor` for the Organization.

#### How should this be manually tested?

A. Test with `statement_descriptor_suffix` set.
- In the admin, set a descriptor on a RevenueProgram.
- Make a donation on a page that is linked to that RevenueProgram
- Go into the Stripe Dashboard and look for the Payment you just made
- In the details for that Payment, you should see a line `Statement descriptor` 
- Its value should be the Account's "Short Statement descriptor" with a `* ` and the value you assigned to `statement_descriptor_suffix`

B. Test without `statement_descriptor_suffix` set.
- In the admin, verify that there is no value on `statement_descriptor_suffix`
- Make a donation on a page that is linked to that RevenueProgram
- Go into the Stripe Dashboard and look for the Payment you just made
- In the details for that Payment, you should see a line `Statement descriptor` 
- Its value should be the Account's full "Statement descriptor"


#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No.
